### PR TITLE
change snap @person  (now can kick author)

### PR DESCRIPTION
**Pull request by jiratQ**
**Review** pls and **GIVE** me answer

1. only line 18 (not kick auther)
2. only line 19 (kick auther)
3. line 18 and 19 (kick auther)
4. else (Pls comment)

### CODE REVIEW

File: src/utils/kick.py [hunk 1]
```diff
@@ -15,7 +15,9 @@ async def random_kick(bot, ctx, user):
       await ctx.send(f'โชคร้ายหน่อยนะ {str(user)} บายย')
       await user.move_to(None)
     else:
-      await ctx.send(f'โชคดีไป {str(user)}')
+      await ctx.send(f'โชคดีไป {str(user)}') #choose only 1 line between 18 or 19 or keep both?
+      await ctx.send(f'มึงโดนแทนละกั้น {str(ctx.author)}')
+      await ctx.author.move_to(None)
```
Comment (namirinz) [inline src/utils/kick.py:20]: Does it work?

Comment (namirinz) [inline src/utils/kick.py:20]: Does it really work?

Comment (jiratQ) [reply src/utils/kick.py:20]: Yes

**Review by namirinz — APPROVED** [review]
Approved!
Keep both 18-19 lines.

File: src/utils/kick.py [hunk 2]
```diff
@@ -15,7 +15,9 @@ async def random_kick(bot, ctx, user):
       await ctx.send(f'โชคร้ายหน่อยนะ {str(user)} บายย')
       await user.move_to(None)
     else:
-      await ctx.send(f'โชคดีไป {str(user)}')
+      await ctx.send(f'โชคดีไป {str(user)}') #choose only 1 line between 18 or 19 or keep both?
+      await ctx.send(f'มึงโดนแทนละกั้น {str(ctx.author)}')
```
Comment (phusitsom) [inline src/utils/kick.py:19]: Nice karma message

**Review by phusitsom — APPROVED** [review]
Approved

**Comment by phusitsom** [comment]
Am I too slow? :(

**Pull request merged**